content/registry/bfe-opentracing

```diff
@@ -0,0 +1,14 @@
+---
+title: bfe-opentracing
+registryType: instrumentation
+tags:
+  - opentracing
+  - bfe
+  - golang
+  
+repo: https://github.com/baidu/bfe/tree/master/bfe_modules/mod_trace
```

Comment:
I suggest adding a README there, there seems to be no documentation how to use this module.

Reply:
Done